src/layouts/Resources.jsx

```diff
@@ -50,16 +26,19 @@ const ResourcePages = ({
     {resourcePages.length > 0
       ? (
         <>
-          {resourcePages.map((resourcePage, resourceIndex) => (
-            <ResourceCard
-              category={resourcePage.category}
-              fileName={resourcePage.fileName}
-              siteName={siteName}
+          {resourcePages.map((resourcePage, resourceIndex) => {
+            const { fileName, category }= resourcePage
+            const { title, date } = retrieveResourceFileMetadata(fileName);
+            return (
+            <OverviewCard
```

Comment:
Missing `siteName` prop

Reply:
We should add a key for `OverviewCard` as well

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/184/commits/e36e346e1ad587c3a3ca7e2a97f464165de637ca